src/scenes/UserGalleryPage/UserGalleryHeader.tsx

```diff
@@ -81,39 +75,41 @@ function UserGalleryHeader({
     return badges.filter((badge) => badge && badge?.imageURL);
   }, [badges, isArtGobblersEnabled]);
 
+  const [showMore, setShowMore] = useState(false);
+  const handleBioClick = useCallback(() => {
+    setShowMore((previous) => !previous);
+  }, []);
+
   return (
     <StyledUserGalleryHeader gap={2}>
+      {isMobile ? (
+        <StyledUsernameMobile>{username}</StyledUsernameMobile>
+      ) : (
+        <StyledUsername>{username}</StyledUsername>
+      )}
+
       <HStack align="flex-start" justify="space-between">
-        <HStack align="center" gap={8}>
-          {isMobile ? (
-            <StyledUsernameMobile>{displayName}</StyledUsernameMobile>
-          ) : (
-            <StyledUsername>{displayName}</StyledUsername>
-          )}
+        <HStack align="center" gap={8} grow>
+          <StyledUserDetails>
+            {is3ac ? (
+              <ExpandableBio text={unescapedBio} />
+            ) : (
+              <StyledBioWrapper showMore={showMore} onClick={handleBioClick}>
+                <Markdown text={unescapedBio} />
+              </StyledBioWrapper>
+            )}
+          </StyledUserDetails>
+
+          {/* TODO(Terence): Test how this looks w/ badges */}
```

Comment:
Who has a badge?

Reply:
art gobblers will be the first one!

i'd say easiest way to mock this would be to hard-core a badge URL to use on the frontend: https://storage.googleapis.com/gallery-prod-325303.appspot.com/gallery-badges/art-gobblers.png